database/migrations/2017_04_07_231938_create_product_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateProductTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('Product', function (Blueprint $table) {
```

Comment:
The columns in camelCase here should be rewritten as snake_case. `catalogPage` becomes `catalog_page`, etc.